src/buttons/modTicket.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  ButtonInteraction,
+  ChannelType,
+  Client,
+  EmbedBuilder,
+  TextChannel,
+} from 'discord.js'
+import { channelIds, roleIds } from '../globals'
+import { emoji } from '../utils/emojis'
+import { errorEmbed, successEmbed } from '../utils/embeds'
+
+export const button = {
+  name: 'modTicket',
+}
+
+export const execute = async (
+  _client: Client,
+  interaction: ButtonInteraction
+) => {
+  if (!interaction.inCachedGuild()) return
+
+  const auctionsTickets = interaction.client.channels.cache.get(
+    channelIds.modTickets
+  ) as TextChannel
+  const openTicket =
+    (await auctionsTickets.threads.fetchActive()).threads.filter(
+      (t) => t.name === `${interaction.user.username}`
+    ).size >= 1
+      ? true
+      : false
```

Comment:
don't see the need for the tenary operator. `(await auctionsTickets.threads.fetchActive()).threads.filter(
      (t) => t.name === '${interaction.user.username}'
    ).size >= 1` returns a boolean already